spec/models/mentor_spec.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: mentors
+#
+#  id         :uuid             not null, primary key
+#  first_name :string           not null
+#  last_name  :string           not null
+#  trn        :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  school_id  :uuid             not null
+#
+# Indexes
+#
+#  index_mentors_on_school_id          (school_id)
+#  index_mentors_on_trn_and_school_id  (trn,school_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (school_id => schools.id)
+#
+require "rails_helper"
+
+RSpec.describe Mentor, type: :model do
+  context "associations" do
+    it { should belong_to(:school) }
+  end
+
+  context "validations" do
+    subject { create(:mentor) }
```

Comment:
Let's avoid creating in unit tests unless it's required.

```suggestion
```

I _think_ the default subject for a model test is a mentor instance. If not, let's make it a `build` call instead.

```suggestion
    subject { build(:mentor) }
```